server/auth.go

```diff
@@ -64,6 +69,12 @@ func (h *Handler) LoginPost(c echo.Context) error {
 		sess.Save(c.Request(), c.Response())
 
 		return c.Redirect(http.StatusFound, location)
+	} else if _, ok := err.(*ipa.ErrExpiredPassword); ok {
+		log.WithFields(log.Fields{
+			"uid":		    uid,
+			"ipa_client_error": err,
+		}).Info("Password expired, redirecting to /auth/change")
+		return c.Redirect(http.StatusFound, Path(fmt.Sprintf("/auth/change?uid=%s", uid)))
```

Comment:
We probably should protect `/auth/change` with a cookie. So set a value in the cookie here then redirect. The `/auth/change` handlers will then fetch the uid from the cookie.